src/catalogue/aws/src/main/java/catalogue/AwsApplication.java

```diff
@@ -0,0 +1,13 @@
+package catalogue;
+
+import io.micronaut.context.env.Environment;
+import io.micronaut.runtime.Micronaut;
+
+public class AwsApplication {
+    public static void main(String[] args) {
+        Micronaut.build(args)
+                .mainClass(AwsApplication.class)
+                .defaultEnvironments(Environment.AMAZON_EC2)
```

Comment:
This environment will be deduced automatically when deploying in AWS.

Anyway we will disable deduction by default in Micronaut 4, so you may want to disable it here, and then activate the environment via `MICRONAUT_ENVIRONMENTS` as said in the other comment.